index.html

```diff
@@ -3,7 +3,487 @@
+              <label><b>Alexis Holman</b> 3 years ago </label><br />
+              <label
+                >It's funny because the cop and suspect are brothers in real
+                life</label
+              >
+            </div>
+          </div>
+          <div class="likesAndDislikes">
+            <div class="likes">
+              <img src="images/thumbs_up.PNG" alt="likes" />
+              <label>1.1k</label>
+            </div>
+            <div class="dislike">
+              <img src="images/dislike.PNG" alt="dislike" />
+              <label>SHARE</label>
+            </div>
+          </div>
+          <div class="linkSection">
+            <img src="images/dropDown.PNG" alt="drop down" />
+            <a href="#">
+              View 63 replies
+            </a>
+          </div>
+          <div class="individualComment">
+            <div class="individualProfile">
+              <img src="images/Profile.jpg" alt="profile oto"  />
+            </div>
+            <div class="singleComment">
+              <label><b>Skyler Smith</b> 4 years ago </label><br />
+              <label
+                >One of the best story telling hip hop songs and videos ever.
+                This is like a 5 minute movie set to music</label
+              >
+            </div>
+          </div>
+          <div class="likesAndDislikes">
+            <div class="likes">
+              <img src="images/thumbs_up.PNG" alt="likes" />
+              <label>1.1k</label>
+            </div>
+            <div class="dislike">
+              <img src="images/dislike.PNG" alt="dislike" />
+              <label>SHARE</label>
+            </div>
+          </div>
+          <div class="linkSection">
+            <img src="images/dropDown.PNG" alt="drop down" />
+            <a href="#">
+              View 63 replies
+            </a>
+          </div>
+          <div class="individualComment">
+            <div class="individualProfile">
+              <img src="images/Profile.jpg" alt="profile icon"  />
+            </div>
+            <div class="singleComment">
+              <label><b>Skyler Smith</b> 4 years ago </label><br />
+              <label
+                >Nas's verses should be published next to Shakespeare</label
+              >
+            </div>
+          </div>
+          <div class="likesAndDislikes">
+            <div class="likes">
+              <img src="images/thumbs_up.PNG" alt="likes" />
+              <label>1.1k</label>
+            </div>
+            <div class="dislike">
+              <img src="images/dislike.PNG" alt="dislike" />
+              <label>SHARE</label>
+            </div>
+          </div>
+          <div class="linkSection">
+            <img src="images/dropDown.PNG" alt="drop down" />
+            <a href="#">
+              View 63 replies
+            </a>
+          </div>
+        </section>
+      </div>
+      <div class="generalRightSection">
+        <div class="rightSect">
```

Comment:
Let us improve semantics by using ```<aside>``` instead of ```<div class="generalRightSection">```

Reply:
Replaced `<div class="generalRightSection">` with `<aside>`